### PR TITLE
Revamp the `Wallet` page

### PR DESCRIPTION
**Pull request by Artem-Semenov-dev**
This PR changes the design of the "Wallet" page and the menu of the application.
General view of the "Wallet" page and menu: 
image
View of the dialog window: 
image
View of the snack bar with error message:
image

### CODE REVIEW

File: client/src/main/kotlin/io/spine/examples/shareaware/client/wallet/WalletPage.kt [hunk 1]
```diff
@@ -335,35 +337,38 @@ public class WalletPageModel(private val client: DesktopClient) {
...
-            horizontalArrangement = Arrangement.Center,
-            verticalAlignment = Alignment.Bottom
-        ) { BalanceCard(model) }
-        Row(
-            modifier = Modifier
-                .weight(1f)
-                .fillMaxWidth(),
-            horizontalArrangement = Arrangement.SpaceAround
+                .fillMaxSize()
+                .zIndex(0f),
```
Comment (Vladyslav-Kuksiuk) [inline client/src/main/kotlin/io/spine/examples/shareaware/client/wallet/WalletPage.kt:351]: Is it necessary?

File: client/src/main/kotlin/io/spine/examples/shareaware/client/wallet/WalletPage.kt [hunk 2]
```diff
@@ -335,35 +337,38 @@ public class WalletPageModel(private val client: DesktopClient) {
...
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        containerColor = MaterialTheme.colorScheme.surface,
+        bottomBar = { PaymentError(model) }
     ) {
-        Row(
+        Column(
```
Comment (Vladyslav-Kuksiuk) [inline client/src/main/kotlin/io/spine/examples/shareaware/client/wallet/WalletPage.kt:348]: It may be better to use `Box` here.

**Review by Vladyslav-Kuksiuk — COMMENTED** [review]
Mostly LGTM with some comments.

File: client/src/main/kotlin/io/spine/examples/shareaware/client/Navigation.kt [hunk 3]
```diff
@@ -80,37 +85,49 @@ public fun MenuLayout() {
...
-                        painterResource(page.iconPath),
-                        contentDescription = page.label,
-                        modifier = Modifier
-                            .width(24.dp)
-                            .height(24.dp)
+                    .width(150.dp)
+                    .height(30.dp)
+                    .clip(MaterialTheme.shapes.extraSmall)
+            ) {
+                NavigationRailItem(
```
Comment (armiol) [inline client/src/main/kotlin/io/spine/examples/shareaware/client/Navigation.kt:97]: Let's reduce the nesting by extracting this bit.

**Review by armiol — APPROVED** [review]
@Artem-Semenov-dev LGTM with a comment to address.

@Vladyslav-Kuksiuk @Artem-Semenov-dev in general, when creating UI layout, let's aim for less indented and less nested code. It's always better to have another private type (or function, at the very least), since it both improves the readability of the original piece, and gives us more opportunity to name _and_ document things we extract.

**Pull request merged**